app/views/posts/_all_posts.html.erb

```diff
@@ -0,0 +1,7 @@
+<% posts.each do |post| %>
+    <%= post.body %><br>
+    <% if current_user == post.user %>
+        <%= link_to  "Delete", post_path(post.id), method: :delete %> | 
+        <%= link_to  "Edit", edit_post_path(post.id) %><br><br>
```

Comment:
- [ ] `homes_controller` `index` action already sends `current_user` posts. So we don't need to check if the `current_user` is the post user.